src/modules/end-user-guide/pages/create-application.adoc

```diff
@@ -0,0 +1,245 @@
+  "annotations": []
+}
+----
+
+== Creating an application
+
+To create a new application, follow the steps below.
+
+Access your {PRODUCT_NAME} instance and click the btn:[Admin Area] application.
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "application/c-application.png",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "annotations": [{
+    "kind": "box",
+    "x": 967,
+    "y": 356,
+    "w": 204,
+    "h": 166
+  }]
+}
+----
+
+Locate and click on the menu:System[Applications] button.
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "application/application-button.png",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "annotations": [{
+    "kind": "box",
+    "x": 13,
+    "y": 179,
+    "w": 161,
+    "h": 24
+  }]
+}
+----
+
+Locate and click on the btn:[new] button. 
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "application/new-application.png",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "annotations": [{
+    "kind": "box",
+    "x": 1369,
+    "y": 64,
+    "w": 50,
+    "h": 24
+  }]
+}
+----
+
+Complete the form with the application's name and click the btn:[submit] button.
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "application/create-application.png",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "annotations": [{
+    "kind": "box",
+    "x": 1451,
+    "y": 216,
+    "w": 64,
+    "h": 25
+  }]
+}
+----
+
+== Editing an application.
+
+To make changes to the details of an application, Access your {PRODUCT_NAME} instance and click the btn:[Admin Area] application.
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "application/c-application.png",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "annotations": [{
+    "kind": "box",
+    "x": 967,
+    "y": 356,
+    "w": 204,
+    "h": 166
+  }]
+}
+----
+
+Locate and click on the menu:System[Applications] button.
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "application/application-button.png",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "annotations": [{
+    "kind": "box",
+    "x": 13,
+    "y": 179,
+    "w": 161,
+    "h": 24
+  }]
+}
+----
+
+Locate and click on the btn:[Edit] icon. 
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "application/new-edit-application.png",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "annotations": [{
+    "kind": "box",
+    "x": 1487,
+    "y": 189,
+    "w": 40,
+    "h": 24
+  }]
+}
+----
+
+Make the necessary changes to the form. 
+You might, for instance, modify the name or replace the application's thumbnail.
+
+[annotation,role="data-zoomable"]
+----
+{
+  "image": {
+    "rel": "application/edit-application.png",
+    "w": 1920,
+    "h": 1080
+  },
+  "view": {},
+  "annotations": []
+}
+----
+
+Additionally, you can enable the visibility of the newly created by clicking on the list checkbox. 
```

Comment:
I think we should move everything regarding visibility to a new section like how there is a dedicated section for archiving [here](https://docs.cortezaproject.org/corteza-docs/2022.9/integrator-guide/security-model/users.html)